avogadro/renderingdialog.ui

```diff
@@ -1,170 +1,239 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <ui version="4.0">
```

Comment:
The file was not formatted actually as per my vs code formatter. Also, this file may have merge conflicts with the blur one. So, after merging any of the PR, I will resolve the merge conflicts in other file.